[Bug][Manager] Failed to start the sort task because the es hosts parameter is incorrect

### What happened

Failed to start the sort task because the es connector `hosts` parameter is incorrect.

### What you expected to happen

```java
Table options are:

'connector'='elasticsearch-6-inlong'
'document-type'='null'
'hosts'='10.x.x.x'
'index'='test_index3'
'inlong.metric.labels'='groupId=group_05&streamId=stream_01&nodeId=sink_01'
'metrics.audit.proxy.hosts'='x.x.x.x:10081'
'password'='xxxxx'
'username'='xxxxx'
        at org.apache.flink.table.factories.FactoryUtil.createTableSink(FactoryUtil.java:171) ~[flink-table-common-1.13.5.jar:1.13.5]
        at org.apache.flink.table.planner.delegation.PlannerBase.getTableSink(PlannerBase.scala:367) ~[flink-table-planner-blink_2.11-1.13.5.jar:1.13.5]
        at org.apache.flink.table.planner.delegation.PlannerBase.translateToRel(PlannerBase.scala:201) ~[flink-table-planner-blink_2.11-1.13.5.jar:1.13.5]
        at org.apache.flink.table.planner.delegation.PlannerBase$$anonfun$1.apply(PlannerBase.scala:162) ~[flink-table-planner-blink_2.11-1.13.5.jar:1.13.5]
        at org.apache.flink.table.planner.delegation.PlannerBase$$anonfun$1.apply(PlannerBase.scala:162) ~[flink-table-planner-blink_2.11-1.13.5.jar:1.13.5]
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234) ~[scala-library-2.11.12.jar:?]
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234) ~[scala-library-2.11.12.jar:?]
        at scala.collection.Iterator$class.foreach(Iterator.scala:891) ~[scala-library-2.11.12.jar:?]
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1334) ~[scala-library-2.11.12.jar:?]
        at scala.collection.IterableLike$class.foreach(IterableLike.scala:72) ~[scala-library-2.11.12.jar:?]
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54) ~[scala-library-2.11.12.jar:?]
        at scala.collection.TraversableLike$class.map(TraversableLike.scala:234) ~[scala-library-2.11.12.jar:?]
        at scala.collection.AbstractTraversable.map(Traversable.scala:104) ~[scala-library-2.11.12.jar:?]
        at org.apache.flink.table.planner.delegation.PlannerBase.translate(PlannerBase.scala:162) ~[flink-table-planner-blink_2.11-1.13.5.jar:1.13.5]
        at org.apache.flink.table.api.internal.TableEnvironmentImpl.translate(TableEnvironmentImpl.java:1518) ~[flink-table-api-java-1.13.5.jar:1.13.5]
        at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeInternal(TableEnvironmentImpl.java:740) ~[flink-table-api-java-1.13.5.jar:1.13.5]
        at org.apache.flink.table.api.internal.StatementSetImpl.execute(StatementSetImpl.java:99) ~[flink-table-api-java-1.13.5.jar:1.13.5]
        at org.apache.inlong.sort.parser.result.FlinkSqlParseResult.executeLoadSqls(FlinkSqlParseResult.java:82) ~[?:?]
        at org.apache.inlong.sort.parser.result.FlinkSqlParseResult.execute(FlinkSqlParseResult.java:61) ~[?:?]
        at org.apache.inlong.sort.Entrance.main(Entrance.java:72) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_151]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_151]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_151]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_151]
        at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:355) ~[flink-clients_2.11-1.13.5.jar:1.13.5]
        ... 12 more
Caused by: org.apache.flink.table.api.ValidationException: Could not parse host 'x.x.x.x' in option 'hosts'. It should follow the format 'http://host_name:port'.
        at org.apache.inlong.sort.elasticsearch6.table.Elasticsearch6Configuration.validateAndParseHostsString(Elasticsearch6Configuration.java:75) ~[?:?]
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_151]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1380) ~[?:1.8.0_151]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_151]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_151]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_151]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_151]
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_151]
        at org.apache.inlong.sort.elasticsearch6.table.Elasticsearch6Configuration.getHosts(Elasticsearch6Configuration.java:44) ~[?:?]
        at org.apache.inlong.sort.elasticsearch6.table.Elasticsearch6DynamicSinkFactory.validate(Elasticsearch6DynamicSinkFactory.java:120) ~[?:?]
```

### How to reproduce

Configure writing to es on the dashboard. Then, this problem will occur during initialization.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java
+++ b/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java
@@ -429,6 +429,8 @@ public class LoadNodeUtils {
      */
     public static ElasticsearchLoadNode createLoadNode(ElasticsearchSink elasticsearchSink,
             List<FieldInfo> fieldInfos, List<FieldRelation> fieldRelations, Map<String, String> properties) {
+        String host = elasticsearchSink.getHost();
+        Integer port = elasticsearchSink.getPort();
         return new ElasticsearchLoadNode(
                 elasticsearchSink.getSinkName(),
                 elasticsearchSink.getSinkName(),
@@ -439,7 +441,7 @@ public class LoadNodeUtils {
                 null,
                 properties,
                 elasticsearchSink.getIndexName(),
-                elasticsearchSink.getHost(),
+                String.format("http://%s:%d", host, port),
                 elasticsearchSink.getUsername(),
                 elasticsearchSink.getPassword(),
                 elasticsearchSink.getDocumentType(),

